### PR TITLE
Native snippets component v1

### PR DESCRIPTION
**Pull request by danbzuro**

### CODE REVIEW

File: src/renderer/components/snippets/NativeSnippetFinder/NativeSnippetFooter.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="absolute bottom-0 left-0 right-0 flex" :class="{'border-t':currentView === 'NewSnippet'}">
+
+    <button @click="changeView('Home')" v-if="currentView === 'NewSnippet'" class="p-4 w-1/2 text-sm text-gray-400">Save
+      and close
+    </button>
+    <button @click="changeView('Home')" v-if="currentView === 'NewSnippet'"
+            class="p-4 w-1/2 text-sm border-l text-gray-600">Save and Add New
+    </button>
+    <button style="background-color: #453FD3" v-if="settings === true"
```
Comment (JuanNitram) [inline src/renderer/components/snippets/NativeSnippetFinder/NativeSnippetFooter.vue]: Is there a possibility to create a specific color class for this on the tailwind config file?

`background-color: #453FD3`

File: src/renderer/components/snippets/NativeSnippetFinder/partials/NativeSnippetHeader.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="p-5 bg-indigo text-white rounded-t-md flex items-center justify-between relative z-20">
+    <button @click="changeView('NewSnippet')" style="background-color: #453FD3" v-if="currentView === 'Home'"
```
Comment (JuanNitram) [inline src/renderer/components/snippets/NativeSnippetFinder/partials/NativeSnippetHeader.vue:3]: Same as before

`background-color: #453FD3`

File: src/renderer/components/snippets/NativeSnippetFinder/NativeSnippetFinder.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  </div>
+</template>
+
+<script>
+  import {mapState} from "vuex";
+  import NativeSnippetHeader from "./partials/NativeSnippetHeader";
+  import NativeSnippetFooter from "./NativeSnippetFooter";
+  import SnippetsList from "./partials/SnippetsList";
+  import NewSnippet from "./partials/NewSnippet";
+  import SnippetsSettings from "./partials/SnippetsSettings";
```
Comment (JuanNitram) [inline src/renderer/components/snippets/NativeSnippetFinder/NativeSnippetFinder.vue]: Please, use aliases here.

File: src/renderer/components/snippets/NativeSnippetFinder/partials/SnippetsSettings.vue [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      <div class="flex justify-between">
+        <span class="font-semibold">Show Notifications</span>
+        cmd+t
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import Toggle from "./Toggle";
```
Comment (JuanNitram) [inline src/renderer/components/snippets/NativeSnippetFinder/partials/SnippetsSettings.vue]: Use alias here.

**Comment by danbzuro** [comment]
@JuanNitram Listo!

**Pull request merged**